keras_cv/layers/preprocessing/with_mixed_precision_test.py

```diff
@@ -145,6 +147,10 @@
 ]
 
 
+@pytest.mark.skipif(
+    multi_backend(),
+    reason="Multi-backend Keras doesn't yet support mixed precision",
```

Comment:
What's the blocker?

Reply:
This wasn't fully working yet (at least it seemed so) when I first ported these, but I just ran all the tests ant it looks good so I've re-enabled these.